NDTensors/src/lib/GradedAxes/src/gradedunitrange.jl

```diff
@@ -80,6 +80,10 @@ function gradedrange(lblocklengths::AbstractVector{<:Pair{<:Any,<:Integer}})
   return gradedrange(labelled.(last.(lblocklengths), first.(lblocklengths)))
 end
 
+function chain(a::GradedUnitRange, b::GradedUnitRange)
```

Comment:
Maybe `axis_cat`? I.e. because it is concatenating one axis with another one, and in Julia `cat` is used for array concatenation.

Also this could be a generic function, for example:
```julia
# Generic function for concatenating axes with blocks.
function blockedunitrange_axis_cat(a::AbstractUnitRange, b::AbstractUnitRange)
  return blockedunitrange(vcat(blocklengths(a), blocklengths(b)))
end

axis_cat(a::GradedUnitRange, b::GradedUnitRange) = blockedunitrange_axis_cat(a, b)

axis_cat(a::BlockedUnitRange, b::BlockedUnitRange) = blockedunitrange_axis_cat(a, b)

# Assume in general there aren't blocks.
# Probably should check the axes are one-based.
axis_cat(a::AbstractUnitRange, b::AbstractUnitRange) = Base.OneTo(length(a) + length(b))
```